specification/synapse/data-plane/Microsoft.Synapse/stable/2023-01-01/linkConnections.json

```diff
@@ -0,0 +1,1011 @@
+          "description": "Continuation token to query table status"
+        }
+      }
+    },
+    "LinkTableStatus": {
+      "type": "object",
+      "description": "The status of a table",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ID provided by the client"
+        },
+        "status": {
+          "type": "string",
+          "description": "Link table status, please refer to this [articles](https://learn.microsoft.com/azure/synapse-analytics/synapse-link/sql-database-synapse-link#monitoring) for details."
+        },
+        "errorMessage": {
+          "type": "string",
+          "description": "Link table error message"
+        },
+        "startTime": {
+          "type": "object",
+          "description": "Link table start time"
+        },
+        "stopTime": {
+          "type": "object",
+          "description": "Link table stop time"
+        },
+        "linkTableId": {
+          "type": "string",
+          "description": "Link table ID"
+        },
+        "errorCode": {
+          "type": "string",
+          "description": "Link table error code"
+        },
+        "lastProcessedData": {
+          "type": "string",
+          "description": "Link table last processed data time",
+          "format": "date-time"
+        },
+        "lastTransactionCommitTime": {
+          "type": "string",
+          "description": "Link table last transaction commit time",
+          "format": "date-time"
+        }
+      }
+    },
+    "QueryTableStatusRequest": {
+      "type": "object",
+      "description": "The query request of table(s) status in link connection",
+      "properties": {
+        "maxSegmentCount": {
+          "type": "integer",
+          "description": "Max segment count to query table status",
+          "format": "int32",
+          "maximum": 1000
+        },
+        "continuationToken": {
+          "type": "object",
+          "description": "Continuation token to query table status"
+        }
+      }
+    },
+    "UpdateLandingZoneCredential": {
+      "type": "object",
+      "description": "New landingZone credential for link connection of SQL Server 2022, see [article](https://learn.microsoft.com/azure/synapse-analytics/synapse-link/sql-server-2022-synapse-link#landing-zone)",
+      "properties": {
+        "sasToken": {
+          "$ref": "../2020-12-01/artifacts.json#/definitions/SecureString",
+          "description": "Landing zone's sas token"
+        }
+      }
+    },
+    "LinkConnectionRefreshStatus": {
+      "type": "object",
+      "description": "The link connection refresh status",
+      "properties": {
+        "refreshStatus": {
+          "type": "string",
+          "description": "Link connection refresh status",
+          "readOnly": true
+        },
+        "errorMessage": {
+          "type": "string",
+          "description": "Link connection refresh error message",
+          "readOnly": true
```

Comment:
Don't mark properties as `readOnly` in schemas that are never part of a request body.

Reply:
removed